source/set-mutable.d.ts

```diff
@@ -0,0 +1,37 @@
+import {Except} from './except';
+import {Mutable} from './mutable';
+
+/**
+Create a type that converts the given keys from `readonly` to mutable. The remaining keys are kept as is.
+
+Use-case: You want to define a single model where the only thing that changes is whether or not some of the keys are mutable.
+
+@see [`Mutable`](source/mutable.d.ts)
```

Comment:
This gives the user no context on "why" it's `@see`. It should explain how it relates. The other way too.